bin/pytorch_inference/Main.cc

```diff
@@ -4,104 +4,117 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+#include <api/CIoManager.h>
 #include <core/CBlockingCallCancellingTimer.h>
 #include <core/CLogger.h>
 #include <core/CProcessPriority.h>
 #include <core/CRapidJsonLineWriter.h>
-#include <core/CoreTypes.h>
-
-#include <ver/CBuildInfo.h>
-
-#include <api/CIoManager.h>
-
 #include <seccomp/CSystemCallFilter.h>
+#include <ver/CBuildInfo.h>
 
 #include "CBufferedIStreamAdapter.h"
 #include "CCmdLineParser.h"
+#include "CCommandParser.h"
 
 #include <rapidjson/ostreamwrapper.h>
 #include <torch/script.h>
 
-#include <boost/optional.hpp>
-
 #include <memory>
 #include <string>
 
-// For ntohl
-#ifdef Windows
-#include <WinSock2.h>
-#else
-#include <netinet/in.h>
-#endif
+static const std::string INFERENCE{"inference"};
+static const std::string ERROR{"error"};
```

Comment:
The modern way is to put file scope constants in an unnamed namespace instead of using `static`.

```suggestion
namespace {
const std::string INFERENCE{"inference"};
const std::string ERROR{"error"};
}
```